src/groovy/org/transmartproject/rest/marshallers/OntologyTermSerializationHelper.groovy

```diff
@@ -93,11 +98,21 @@ class OntologyTermSerializationHelper extends AbstractHalOrJsonSerializationHelp
     @Override
     Map<String, Object> convertToMap(OntologyTermWrapper obj) {
         OntologyTerm term = obj.delegate
-        [
+        def result =
+            [
                 name:     term.name,
                 key:      term.key,
                 fullName: term.fullName,
-        ]
+            ]
+
+        Map<OntologyTerm, List<OntologyTermTag>> map = tagsResource.getTags([term] as Set, false)
+        List<OntologyTermTag> tags = map.get(term)
+
+        if (tags && tags.size() > 0) {
+            result.put('metadata', tags.collectEntries { [(it.name): it.description ] })
+        }
```

Comment:
It was not specified in that case. It would be then an empty Map (not a List).
Is there a JSON/HAL standard for this?
In terms of consistency, having always the element seems better, but for the children links for sample (in the parent/children navigation ticket) we omit the element.